src/tsn_adapters/blocks/tn_access.py

```diff
@@ -73,7 +73,7 @@ def client(self):
     def helper_contract_stream_id(self):
         if not self.helper_contract_name:
             raise self.Error("Helper contract name is not set")
-        return generate_stream_id(self.helper_contract_name)
+        return self.helper_contract_name
```

Comment:
if we want to make the block input being the contract ID directly instead of a readable name, this change would be confusing. ain't better to [change it here](https://github.com/trufnetwork/adapters/blob/fix/unableToInsertDataUsingHelperContract/src/tsn_adapters/blocks/tn_access.py#L56) to `helper_contract_id` and use the correct semantics?